numpy/random/generator.pyx

```diff
@@ -3438,6 +3438,10 @@ cdef class Generator:
         # Check preconditions on arguments
         mean = np.array(mean)
         cov = np.array(cov)
+
+        if np.issubdtype(cov.dtype, np.complexfloating):
+            raise NotImplementedError("Complex gaussians are not supported.")
```

Comment:
This only checks the covariance and not the mean - is that deliberate? If so, I think the message could be clearer